### PR TITLE
Cosmetic Adjustments to Zone/Planet/Level Printouts

### PR DESCRIPTION
**Pull request by Alamei**
* Add difficulty display for zones that are shown in planet readout.
* Change difficulty 1 name from "Low" to "Easy" in lookup function for consistency.
* Slight tweaks to colors and capitalization throughout for consistency.

### CODE REVIEW

**Comment by hintswen** [comment]
Difficulties are the threat levels which are low, medium and high.

**Comment by Alamei** [comment]
True, but the the output for the planets in the script has been calling the zones "Hard", "Medium" and "Easy", so I figured they should match. Perhaps I should have made the change in the other direction ("Easy" to "Low" on the planet output)?

File: cheat.php [hunk 1]
```diff
@@ -118,8 +118,8 @@
 	$Zone = $Zone[ 'response' ][ 'zone_info' ];
 
 	Msg(
-		'>> Joined Zone {green}' . $Zone[ 'zone_position' ] .
-		'{normal} on planet {green}' . $BestPlanetAndZone[ 'id' ] .
+		'>> Joined Zone {yellow}' . $Zone[ 'zone_position' ] .
```
Comment (xPaw) [inline cheat.php]: That was colored differently to make it easier to see.

**Comment by Alamei** [comment]
Understood. Reverted the zones on those lines to green.

File: cheat.php [hunk 2]
```diff
@@ -206,9 +206,9 @@ function GetNameForDifficulty( $Zone )
 
 	switch( $Zone[ 'difficulty' ] )
 	{
-		case 2: $Difficulty = 'Medium'; break;
 		case 3: $Difficulty = 'Hard'; break;
-		case 1: $Difficulty = 'Low'; break;
+		case 2: $Difficulty = 'Medium'; break;
+		case 1: $Difficulty = 'Easy'; break;
```
Comment (xPaw) [inline cheat.php]: I actually just changed it to `Low`, so you'll want to rebase your PR.

**Comment by Lyrex** [comment]
Honestly I think the new naming scheme doesn't make too much sense.
IMO it should either be ["High", "Medium", "Low"] or ["Hard", "Medium", "Easy"]. Mixing these both up feels wrong to me.

**Comment by xPaw** [comment]
I changed all the code from Easy to Low, so it is consistent now.

**Comment by Alamei** [comment]
Good catch, Lyrex. PR updated.

**Pull request closed without merging**